app/(frontend)/auth/login/page.tsx

```diff
@@ -3,15 +3,36 @@
 import Image from "next/image"
 import {Button, Input} from "@nextui-org/react";
 import {Query} from "@/app/lib/api/Query";
+import {useRouter} from "next/navigation";
+import {useState} from "react";
+import {ILoginFormError} from "@/app/lib/api/error/ApiError";
+import Link from "next/link";
+import {useCookies} from "react-cookie";
+import {AUTHORIZATION} from "@/app/lib/definitions";
+
+const DAYS = 1
+const MAXAGE = 60 * 60 * 24 * DAYS
+
 
 export default function LoginPage() {
 
+    const router = useRouter()
+    const [state, setState] = useState<ILoginFormError | string>()
+    const [cookies, setCookie, removeCookie] = useCookies([AUTHORIZATION])
+
     const handleSubmit = async (formData: FormData) => {
         const query = new Query('/auth/login')
         const response = await query.withMethod('POST')
             .withBody(formData)
-            .send<{token: string}>()
-        console.log(response.data)
+            .send<{ token: string } & ILoginFormError>()
+
+        if (response.status === 200) {
+            setCookie(AUTHORIZATION, response.data.token, {maxAge: MAXAGE})
+            router.push('/home')
+        } else {
+            setState(response.data.formError || response.data.error)
+            console.log(typeof state, state)
```

Comment:
I assume this log was put here for debugging purposes. Random logs like these should not be in the code. We do not want the user to see something like this in the console.